core/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ _YYYY-MM-DD_
 * Dependency loop error now lists all test cases that formed a cycle
 * `foldGroup` now takes `[b]` instead of `b` as its last argument to allow for custom fold strategies. This is a backwards incompatible change, but you can get the old behavior by applying `mconcat`.
 * Dependencies can now be defined pattern-free with `sequentialTestGroup`. (#343)
+* `PrihntTest` constructor has extre field used to report progress.
```

Comment:
```suggestion
* `PrintTest` constructor has extra field used to report progress.
```
I'd appreciate if you mention that the progress reporting is not ignored now (hooray!) too.